문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,10 @@
+function solution(s) {
+    let arr = s.split('');
+    let sor = arr.sort();
+    let res = sor.reverse();
+    const answer = res.join('');
```

Comment:
```suggestion
   const arr = s.split('');
    const sor = arr.sort();
    const res = sor.reverse();
    return res.join('');
```
변수선언 방식 신경쓰기, 불필요한 변수 선언 줄이기
```
return s.split('').sort().reverse.().join(''); 
```
또한 가능합니다